src/main.ts

```diff
@@ -214,6 +214,14 @@ function main(args: string[]): number {
     return 1;
   }
 
+  if (config.options.module !== ts.ModuleKind.CommonJS) {
+    // This is not an upstream TypeScript diagnostic, therefore it does not go
+    // through the diagnostics array mechanism.
+    console.error(
+        'tsickle converts TypeScript modules to Closure modules, via CommonJS internally. Set tsconfig.js "module": "commonjs"');
+    return 2;
```

Comment:
nit: as long as we don't have any other program specifically check for this return value, I'd just always return `1`, less confusing that way.